recipe/meta.yaml

```diff
@@ -1,46 +1,49 @@
-{% set version = "20160523b" %}
+{% set version = "20170520" %}
 
 package:
   name: texlive-core
   version: {{ version }}
 
 source:
   fn: texlive-{{ version }}-source.tar.xz
-  url: http://mirrors.ctan.org/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
-  sha256: a8b32ca47f0a403661a09e202f4567a995beb718c18d8f81ca6d76daa1da21ed
+  url: http://mirrors.concertpass.com/tex-archive/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
```

Comment:
Thanks! I will modify that.

> I was working at the same PR today!!

Sorry about that. @bgruening made me feel bad that I did not act on #14 for so long :smile: 

PS: you should send a PR adding yourself as a maintainer though :wink: